.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -34,27 +34,26 @@ _What changed calling `firejail --noprofile /path/to/program` in a terminal?_
 
 ### Additional context
 
-_Any other context you have to understand the problem_
+_Any other detail that may help understand/debug the problem_
 
 ### Environment
 
 - Linux distribution and version (e.g. "Ubuntu 20.04" or "Arch Linux")
 - Firejail version (`firejail --version`).
-  If you use a development version also the commit from which firejail was compiled (`git rev-parse HEAD`).
+- If you use a development version, also the commit from which firejail was compiled (`git rev-parse HEAD`).
 
 ### Checklist
 
-- [ ] The issues is caused by firejail (i.e running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).
+- [ ] The issues is caused by firejail (i.e. running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).
 - [ ] I can reproduce the issue without custom modifications (e.g. globals.local).
 - [ ] The program has a profile. (If not, request one in `https://github.com/netblue30/firejail/issues/1139`)
 - [ ] The profile (and redirect profile if exists) hasn't already been fixed [upstream](https://github.com/netblue30/firejail/tree/master/etc).
 - [ ] I have performed a short search for similar issues (to avoid opening a duplicate).
-  - [ ] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.
+- [ ] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.
```

Comment:
The indentation was done by intention :smile:. Because Browser+DRM/U2F are frequent duplicates, there is this check in the checklist which is actually a specific check for duplicates.
```suggestion
  - [ ] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.
```

Reply:
> The indentation was done by intention smile. Because Browser+DRM/U2F are
> frequent duplicates, there is this check in the checklist which is actually a
> specific check for duplicates.

I thought otherwise because it was a sub list with a single item (and I didn't
see any direct relation between the two items), but I get the intent now.